File: Latihan 1/Latihan B/PHP/daftarTim.php
<?php

class daftarTim{

	private $namaTim;
	private $negaraAsal;
	private $tahun;
	private $pemain;

	public function __construct(){
	}

	public function setNamaTim($namaTim)
	{
		$this->namaTim = $namaTim;
	}

	public function getNamaTim()
	{
		return $this->namaTim;
	}

	public function setNegaraAsal($negaraAsal)
	{
		$this->negaraAsal = $negaraAsal;
	}

	public function getNegaraAsal()
	{
		return $this->negaraAsal;
	}

	public function setTahun($tahun)
	{
		$this->tahun = $tahun;
	}

	public function getTahun()
	{
		return $this->tahun;
	}

	public function setPemain($pemain)
	{
		$this->pemain = $pemain;
	}

	public function getPemain()
	{
		return $this->pemain;
	}

}

?>

File: Latihan 1/Latihan B/C++/daftarTim.cpp
class daftarTim
{
    private:
        string namaTim;
        string negaraAsal;
        string tahun;
        string pemain[20];
     
    public:
        daftarTim(){

        }

        void setNamaTim(string namaTim){
            this->namaTim = namaTim;
        }

        string getNamaTim(){
            return this -> namaTim;
        }

        void setNegaraAsal(string negaraAsal){
            this->negaraAsal = negaraAsal;
        }

        string getNegaraAsal(){
            return this->negaraAsal;
        }

        void setTahun(){
            this->tahun = tahun;
        }

        string getTahun(){
            return this->tahun;
        }

        void setPemain(int i, string pemain){
            this->pemain[i] = pemain;
        }

        string getPemain(int i){
            return this->pemain[i];
        }



        ~daftarTim(){

        }
};
